### PR TITLE
Add wiki frontpage to readme

### PR DESCRIPTION
**Pull request by dreamflasher**
Relevant discord communication:

Me: the tutorials are very hidden and would have been nice to have a "help" menu in the editor that links to an overview of all tutorials, also a link from the github repo to that page… I was looking quite some time for the documentation

ArnoldSmith86: Yeah, the main page should be Playing games, Creating games, Contributing, Hosting own server or something like that. With an overview of how to do it and links to appropriate places.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,44 @@
...
+VTT is an open-source, free to use platform for creating and playing games. This project aims to create a virtual tabletop in the browser where you can (re)create all board, dice and card games and play them without registration over the internet.
 
-You can host your own instance or go to [VirtualTabletop.io](https://virtualtabletop.io).
+## Playing games
 
-This project is inspired by [playingcards.io](https://playingcards.io).
+We have over 170 games in our public library that are ready to play right now.  That includes classic games like Chinese checkers, backgammon, mancala, Nine Men’s Morris, Reversi, tangrams, and hangman.  It includes modern games that are similar to many popular card and board games available around the world today.  Most of the games are in English, but quite a few are “universal” and require no ability to read or understand English.  And a handful are offered in German and Portuguese and a few more in Spanish and French. See [Games](https://github.com/ArnoldSmith86/virtualtabletop/wiki/Games) for more detail.
```
[the remark this answers is not in the document]
Comment (dreamflasher) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -1,34 +1,44 @@
...
+This project is inspired by [playingcards.io](https://playingcards.io).
+
+This project aims to be able to import PCIO files and have them be immediately playable.
 
-### Linux
+Come join our [Discord server](https://discord.gg/CEZz7wny9T) if you want to discuss the project.
+
+This project started after [a reddit post](https://www.reddit.com/r/PlayingCardsIO/comments/jiajth/mildly_ot_feature_requests_for_a_pcio_clone/) in our [playingcardsio subreddit](https://www.reddit.com/r/PlayingCardsIO/).
+
+### Download this repository and get all dependencies
```
[the remark this answers is not in the document]
Comment (dreamflasher) [reply README.md:39]: done

**Review by 96LawDawg — APPROVED** [review]
Looks good to me.

**Review by ArnoldSmith86 — APPROVED** [review]
Probably not the best it can be and we should include some screenshots or videos.

But it's already way better than the current version..

**Pull request merged**